controller/aks-cluster-config-handler.go

```diff
@@ -221,6 +221,17 @@ func (h *Handler) createCluster(config *aksv1.AKSClusterConfig) (*aksv1.AKSClust
 		return config, err
 	}
 
+	resourceClusterClient, err := aks.NewClusterClient(credentials)
+	if err != nil {
+		return config, err
+	}
+
+	logrus.Infof("Checking if cluster [%s] exists", config.Spec.ClusterName)
+
+	if aks.ExistsCluster(ctx, resourceClusterClient, &config.Spec) {
+		return config, fmt.Errorf("cluster [%s] already exists in AKS. Please import it", config.Spec.ClusterName)
```

Comment:
```suggestion
		return config, fmt.Errorf("cluster [%s] already exists in AKS", config.Spec.ClusterName)
```

I think we might not want to suggest importing the cluster, if a user is trying to provision a hosted cluster from rancher they likely want rancher to manage the lifecycle.

Reply:
Are you saying that we shouldn't assume the user's end goal in the error msg?